src/client/src/App/Credit/CreditRfqs/CreditRfqCards/styled.tsx

```diff
@@ -14,20 +14,47 @@ export const CreditRfqCardsWrapper = styled.div<{ empty: boolean }>`
   flex: 1;
 `
 
+const cardFlash = ({
+  theme,
+  direction,
+}: {
+  theme: DefaultTheme
+  direction: Direction
+}) => keyframes`
+  0% {
+    border-color: ${theme.colors.spectrum.uniqueCollections[direction].base};
+  }
+  50% {
+    border-color: transparent;
+  }
+  100% {
+    border-color: ${theme.colors.spectrum.uniqueCollections[direction].base};
+  }
+`
+
+const highlightBorderColor = css`
```

Comment:
could you double-check the behaviour on page refresh - for blotter highlight it re-highlights the last highlighted row, which looks a bit daft, but certainly not serious

Reply:
This doesn't seem to happen for the tiles